dialogues/rags/study.json

```diff
@@ -0,0 +1,10 @@
+{
+  "first": {
+    "content": "You enter the parlour in the front of the house. The first thing you notice is a tree branch sticking through the window on the right. There is glass on the carpet. In the center of the room is a coffee table with two arm chairs around it, and across the room is a large bookcase against the wall.&&A sliding door on the left leads into what used to be the dining room.",
```

Comment:
This needs to be fixed. There is an unseen rift inside the window, the the branch is in the stuffy in the other timeline.